StorageTests/TestSQLiteHistoryRecommendations.swift

```diff
@@ -134,6 +134,61 @@ class TestSQLiteHistoryRecommendations: XCTestCase {
         XCTAssertEqual(highlights.count, 1)
         XCTAssertEqual(highlights[0]!.title, "A Bookmark")
     }
+
+    /*
+     * Verify that we do not return a highlight if
+     * its domain is in the blacklist
+     *
+     */
+    func testBlacklistHighlights() {
+        let db = BrowserDB(filename: "browser.db", files: files)
+        let prefs = MockProfilePrefs()
+        let history = SQLiteHistory(db: db, prefs: prefs)
+
+        let startTime = NSDate.nowMicroseconds()
+        let oneHourAgo = startTime - oneHourInMicroseconds
+        let fifteenMinutesAgo = startTime - 15 * microsecondsPerMinute
+
+        /*
+         * Site A: 1 visit, 1 hour ago = highlight
+         * Site B: 1 visits, 15 minutes ago = non-highlight
+         * Site C: 3 visits, 1 hour ago = highlight
+         * Site D: 4 visits, 1 hour ago = non-highlight
+         */
+        let siteA = Site(url: "http://www.google.com", title: "A")
+        let siteB = Site(url: "http://siteB/", title: "B")
+        let siteC = Site(url: "http://www.search.yahoo.com/", title: "C")
```

Comment:
Nit: I would add a comment clarifying that yahoo/google domains are on the blacklist. Without that, it looks like this test doesn't actually do much.